src/main/scala/viper/gobra/frontend/info/implementation/resolution/MemberResolution.scala

```diff
@@ -86,7 +86,7 @@ trait MemberResolution { this: TypeInfoImpl =>
     def go(pastDeref: Boolean): Type => AdvancedMemberSet[AdtMember] = attr[Type, AdvancedMemberSet[AdtMember]] {
 
       case DeclaredT(decl, context) => go(pastDeref)(context.symbType(decl.right)).surface
-      case PointerT(t) if !pastDeref => go(pastDeref = true)(t).ref
+      case ActualPointerT(t) if !pastDeref => go(pastDeref = true)(t).ref
```

Comment:
If we have ghost pointer receiver, then you should duplicate the cases for ActualPointer also for GhostPointer (in this entire file). Doing the duplication should implement the expected member propagation for ghost pointer.